From 94e77b115f5b14b7dfea50e4a7b77047e4b47263 Mon Sep 17 00:00:00 2001
From: Paul Varner <fuzzyray@gentoo.org>
Date: Tue, 18 Dec 2012 04:33:27 -0600
Subject: [PATCH 1/3] Fix setup.py so that either VERSION or PVR will set the
 version.

This fixes the issue with egg-info always having a version number of
9999.  Fix is from Brian Dolbec courtesy of mirrorselect.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 9d2addd..45a4c06 100755
--- a/setup.py
+++ b/setup.py
@@ -14,7 +14,7 @@ import io
 
 sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'pym'))
 
-__version__ = os.getenv('VERSION', default='9999')
+__version__ = os.getenv('VERSION', default=os.getenv('PVR', default='9999'))
 
 cwd = os.getcwd()
 
-- 
1.8.5.4

